CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Update .gitattributes so we have consistent line endings
 - Change 266 files from CRLF to LF.
 - Run tests on push as well as on a pull request so developers can see impact
+- Align value indicators in assertion statements
```

Comment:
Maybe I am being picky, but it would be great if you could split up the current commit (4bf8b53b) into two separate commits, one for each entry here.

Reply:
Can you post a screenshot of the before/after alignment?  I'm not sure what this change is addressing